### DISCUSSION TITLE
Lazyload tabs in CRUDS

### DISCUSSION THREAD

**Discussion by lieto-dk** [Laravel-Backpack/community-forum#885] [Ideas]
Hi,
Is there anyway to "lazyload" tabs in a crud. I have a few Cruds where there is quite a few tabs with subfields and the full crud is not loading as fast as i wish. If i could lazyload each tab when it is activated - that would be great and speed up initial load of the crud.
/jens

**Comment by jcastroa87** [comment]
Hello @lieto-dk 

At this time, we do not have this feature, however, we hope to develop it for future updates, perhaps in Backpack 7.

Cheers.

**Reply by lieto-dk** [reply]
Hi, Jorge,

That sounds great. Do you need me to create an issue or post it somewhere for you guys to pick it up for your development plan ?

**Comment by jcastroa87** [comment]
Don't worry, I will keep it in the ideas category.

Thanks
